pkg/services/ngalert/api/tooling/post.json

```diff
@@ -2904,9 +2918,9 @@
      "$ref": "#/definitions/Userinfo"
     }
    },
-   "title": "A URL represents a parsed URL (technically, a URI reference).",
+   "title": "URL is a custom URL type that allows validation at configuration load time.",
    "type": "object",
-   "x-go-package": "net/url"
+   "x-go-package": "github.com/prometheus/common/config"
```

Comment:
Why did this change?

Reply:
it's autogenerated; I believe because there are two conflicting URLs used: [one defined in prometheus](https://pkg.go.dev/github.com/prometheus/common/config#URL) and [one defined in net/url](https://pkg.go.dev/net/url#URL).
Not sure how to properly solve this.